MAUI/TabView/Tab-Item-Customization.md

```diff
@@ -441,14 +441,12 @@ N> The `TabHeaderPadding` property is only applicable when [TabWidthMode](https:
 {% tabs %}
 
 {% highlight xaml %}
-<tabView:SfTabView>
-    <tabView:SfTabItem TabWidthMode="SizeToContent" TabHeaderPadding="5,10,5,10">
+<tabView:SfTabView TabWidthMode="SizeToContent" TabHeaderPadding="5,10,5,10">
```

Comment:
Why did you remove the tab item? just move the properties to the tab view.

Reply:
Now, I added the tab item.